Godeps/Godeps.json

```diff
@@ -2791,6 +2791,58 @@
 			"Rev": "c2ac40f1adf8c42a79badddb2a2acd673cae3bcb"
 		},
 		{
+			"ImportPath": "k8s.io/metrics/pkg/apis/custom_metrics",
```

Comment:
I'm hitting this circular dependency. kubernetes->metric->client-go (which is in kubernetes/staging).

I'm going to generate the metrics client to untangle part of the dependency.

There are other circular dependencies caused by vendoring k8s.io/metrics. IMO we should push to remove the staging folder and develop in the real repos to break the cycle.

cc @thockin another problem caused by staging.